pkg/pgmodel/metrics/metrics.go

```diff
@@ -37,6 +37,13 @@ var (
 			Name:      "decompress_min_unix_time",
 			Help:      "Earliest decdompression time",
 		}, []string{"table"})
+	DuplicateMetrics = prometheus.NewCounter(
```

Comment:
I would prefer this go under the DuplicateWrites declaration. Keep all the duplicate stuff together